keras_cv/layers/preprocessing/mosaic_test.py

```diff
@@ -80,7 +80,7 @@ def test_return_shapes(self):
 
         self.assertEqual(xs.shape, [2, 512, 512, 3])
         self.assertEqual(ys_labels.shape, [2, 10])
-        self.assertEqual(ys_bounding_boxes.shape, [2, 12, 5])
+        self.assertEqual(ys_bounding_boxes.shape, [2, None, 5])
```

Comment:
any reason this is changed to None?

Reply:
It outputs a ragged instead of a padded dense now after the API change - this is an expected change.

Reply:
(we could call to_tensor() in the test if we want)